cursorless-talon/src/cheatsheet/cheat_sheet.py

```diff
@@ -43,9 +43,9 @@ def cursorless_open_instructions():
 class Actions:
     def cursorless_cheat_sheet_show_html():
         """Show cursorless html cheat sheet"""
-        # NB: We use the user's home directory instead of temp to make sure that
+        # NB: We use the user's documents directory instead of temp to make sure that
         # Linux snaps work
-        cheatsheet_out_dir = Path.home() / ".cursorless" / "cheatsheet"
+        cheatsheet_out_dir = Path.home() / "Documents" / ".cursorless" / "cheatsheet"
```

Comment:
This needs to actually look up what the Documents directory is called, since it can be renamed, or can have a different default name based on locale.

Reply:
Like this? https://github.com/cursorless-dev/cursorless/issues/1266#issuecomment-1429488373

Reply:
Fixed